### PR TITLE
update green ci

### LINKED ISSUE #916: CI green for https://github.com/pawelru/ie_status?tab=readme-ov-file
todos

- [ ] nosuggests https://github.com/insightsengineering/rtables/actions/runs/10437584162/job/28903931146

### PR DESCRIPTION
**Pull request by shajoezhu**
close #916

### CODE REVIEW

**Review by Melkiades — APPROVED** [review]
Lgtm! Thanks Joe

File: tests/testthat/test-as_html.R [hunk 1]
```diff
@@ -1,6 +1,9 @@
 context("Exporting to HTML")
 
 test_that("as_html smoke test", {
+  require(xml2, quietly = TRUE)
+  skip_if_not_installed("xml2")
+
```
Comment (pawelru) [inline tests/testthat/test-as_html.R:6]: In general it's better to return early before executing (independent) commands

```suggestion
  skip_if_not_installed("xml2")
  require(xml2, quietly = TRUE)

```

File: tests/testthat/setup-fakedata.R [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ## Loading relevant libraries for tests
 library(testthat)
```
Comment (pawelru) [inline tests/testthat/setup-fakedata.R:2]: we already have it here: 
```suggestion
```

File: tests/testthat/setup-fakedata.R [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
-library(xml2)
-library(tibble)
+require(tibble, quietly = TRUE)
 library(rtables)
```
Comment (pawelru) [inline tests/testthat/setup-fakedata.R:4]: This is not needed as `testthat` automatically attach tested package namespace (including private functions)
```suggestion
```

File: tests/testthat/test-as_html.R (hunk 1 above)
Comment (Melkiades) [reply tests/testthat/test-as_html.R:6]: @shajoezhu this is what I mentioned in the last PRs ;)

**Pull request merged**